src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.html

```diff
@@ -7,7 +7,7 @@
              [ngClass]="{'d-flex': flexHeight}"
              aria-label="Details card">
       <dl class="ms-4 me-4">
-        <dt>FSID</dt>
+        <dt>Cluster ID</dt>
```

Comment:
maybe a helper which says this is the FSID of the cluster?

Reply:
Sure, can you verify if it looks good?
```
              The Cluster ID is same as File System Identifier (fsid)
              and it is a unique identifier for the cluster.
```

Reply:
just to mention now `ceph -s` command calls it cluster id.

```
  cluster:
    id:     58918bad-6955-4fe7-be29-300a8b12e385
    health: HEALTH_WARN
```